bindings/wasm/examples/public/examples.js

```diff
@@ -51,6 +51,34 @@ export const examples = {
       return result;
     },
 
+    Auger: function() {
```

Comment:
Thanks for the example! But I'm also really picky about public examples, so I went ahead and made my own instead. Please don't feel slighted; it's just my own aesthetics.